dbms/tests/server-test.xml

```diff
@@ -75,6 +75,20 @@
                 </replica>
             </shard>
         </test_shard_localhost>
+        <test_cluster_two_shards>
```

Comment:
Not so easy :D 

To make it work in CI you need to do smth like:
https://github.com/ClickHouse/ClickHouse/blob/master/docker/test/stateless/Dockerfile#L39
and rebuild corresponding docker image.
And don't ask me why do i know that :)

Reply:
I already added this cluster into [config.xml](https://github.com/ClickHouse/ClickHouse/blob/master/dbms/programs/server/config.xml), this should be enough I guess? (I don't see any harm in adding this into config, do you?)

And, yeah, as far as I figured out, `server-test.xml` does not used for stateless tests, so I guess that the primary one is used.

Reply:
It should get the config, but not sure if the config having the same host few times will be accepted as valid.

Reply:
>It should get the config

Great, thanks!

>but not sure if the config having the same host few times will be accepted as valid.

It will, but anyway the hosts are different, 127.0.0.2 and 127.0.0.1